infra/feast-operator/test/testdata/feast_integration_test_crs/v1alpha1_default_featurestore.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: feast.dev/v1alpha1
+kind: FeatureStore
+metadata:
+  name: default-feast-cr
+spec:
+  feastProject: my_project
+  services:
+    registry:
```

Comment:
we should test all 3 service types with this PR... onlineStore and offlineStore should be added

Reply:
Sure. I will add those changes to PR as well. It needs its own asserts and some refactoring of the code is needed to make it generic asserts. I will continue working on it.

Reply:
Updated the code to reflect this change. PTAL.